README.md

```diff
@@ -1,15 +1,26 @@
 # Rummager
 
-Rummager is now primarily based on elasticsearch.
+Rummager is the internal GOV.UK API for search.
 
-## Get started
+![Screenshot of API Response](docs/api-screenshot.png)
```

Comment:
I'm not sure a screenshot is that useful for an API-only service.  It may make sense to include an example of the json returned for a given request, but not as an image...